dpe/src/commands/sign.rs

```diff
@@ -20,20 +20,25 @@ pub struct SignCmd {
 
 impl SignCmd {
     const _IS_SYMMETRIC: u32 = 1 << 31;
-    const _ND_DERIVATION: u32 = 1 << 30;
+    const ND_DERIVATION: u32 = 1 << 30;
 
     const fn _uses_symmetric(&self) -> bool {
         self.flags & Self::_IS_SYMMETRIC != 0
     }
 
     /// Uses non-deterministic derivation. If symmetric algorithms are used, this flag is ignored.
-    const fn _uses_nd_derivation(&self) -> bool {
-        !self._uses_symmetric() && self.flags & Self::_ND_DERIVATION != 0
+    const fn uses_nd_derivation(&self) -> bool {
+        !self._uses_symmetric() && self.flags & Self::ND_DERIVATION != 0
```

Comment:
nit: Now that this function is used we can also remove the underscore prefix from `uses_symmetric`. But could/should be a separate PR.